Task 2/2.2/Static.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _2._2
+{
+    public abstract class Static : Objects
```

Comment:
StaticObjects. Names of types or varaibles should be nouns (or at least that should be the main word)